test/helpers/parallelizable_reporter.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module Minitest
+  module Reporters
+    class ParallelizableReporter < DefaultReporter
+      def on_record(test)
+        # do not print the marks if test names are also printed
+        # paralellism causes the names and results to be mismatched
+        unless options[:verbose]
+          # Print the pass/skip/fail mark
+          result = if test.passed?
+            record_pass(test)
+          elsif test.skipped?
+            record_skip(test)
+          elsif test.failure
+            record_failure(test)
+          end
+          print(result)
+        end
+
+        # Print fast_fail information
+        return unless @fast_fail && test.failure # test.failure includes skips
+        return if test.skipped? && !@detailed_skip
+        puts
+        puts
```

Comment:
Two puts in a row is odd. Is this code copied from somewhere?

Reply:
[Kinda](https://github.com/kern/minitest-reporters/blob/master/lib/minitest/reporters/default_reporter.rb), yes (I started by copying things and then changed most of it). I could `puts "\n"` instead.

Reply:
I think that would be better. When I see something like this, I start wondering if its a work around or a bug or something.